lib/web_ui/dev/firefox.dart

```diff
@@ -48,12 +48,11 @@ class Firefox extends Browser {
       var dir = createTempDir();
       var args = [
         url.toString(),
-        if (!debug) '--headless',
+        '--headless',
```

Comment:
I'm assuming this is because we do not support debugging in Firefox? If so, let's throw an exception if `debug` is `true` on Firefox.

Reply:
No it is not. 

I am moving to the headless mode due to excessive number of popups (default browser, old version, crash, safe mode). Unfortunately there is not any single preference to block them all, and there are so many added in different versions.